lib-opensource/emqx_management/src/emqx_mgmt.erl

```diff
@@ -554,6 +556,17 @@ list_listeners(Node) when Node =:= node() ->
 list_listeners(Node) ->
     rpc_call(Node, list_listeners, [Node]).
 
+restart_listener(false) ->
+    {error, "No such listener"};
+restart_listener(Listener) ->
+    emqx_listeners:restart_listener(Listener).
+
+restart_listener(Node, Listener) when Node =:= node() ->
```

Comment:
This is an RPC call handler API, the Identifyer -> Listener lookup should be done here.
suggestion: move the lookup to emqx_listeners.erl.

i.e. emqx_listeners:restart_listener/1 accepts both a #{} = Listener and a Identifier of type string() | binary().